### PR TITLE
ganyoa - adding lesson06 assignment

### PR DESCRIPTION
**Pull request by ganyoa**
Hi, please see uploaded lesson06 mailroom and mailroon test assignments.  Thank you, Anthony Ganyo

### CODE REVIEW

File: students/ganyoa/lesson06/mailroom_lesson06.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
+import sys
+import operator
+import os
+
+#dictionary of current donors and donation amounts
+donor_dict = {"Mouth Devereaux" : [4562.58, 3817.39],
```
Comment (charlesrobison) [inline students/ganyoa/lesson06/mailroom_lesson06.py]: No white space after keys

File: students/ganyoa/lesson06/mailroom_lesson06.py [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+            "Mikey Walsh" : [618.45, 49.20, 542.87],
+            "Andy Carmichael" : [75.28, 99.25],
+            "Chester Copperpot" : [750254.00, 802154.11, 1.75]
+            }
+
+def new_donor_dict_func():
+    #creates new dict with donation amnt summed, counted, and averaged; sorted by total
+    new_donor_dict = {x: (sum(y),len(y),average(y)) for (x, y) in donor_dict.items()}
```
Comment (charlesrobison) [inline students/ganyoa/lesson06/mailroom_lesson06.py]: Missing white space after (y), in both places

File: students/ganyoa/lesson06/mailroom_lesson06.py [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+
+def ty_letter(a, b):
+    #thank you letter
+    return ('\n'.join(('\n''Dear {},\n',
+    'Thank you for your generous donation of ${:.2f}.',
+    'Your commitment goes a long way in helping us',
+    'solve things that need to be solved.' + '\n',
+    'Warm regards,' + '\n',
+    'Mama Fratelli')).format(a,b))
```
Comment (charlesrobison) [inline students/ganyoa/lesson06/mailroom_lesson06.py]: Missing white space here too, after a,

**Comment by ganyoa** [comment]
Hi, am I able to add additional pull requests, or does this one have to be closed first?  Thank you, Anthony Ganyo

**Comment by natasha-aleksandrova** [comment]
@ganyoa is your lesson 7 complete?

you can only have one open at a time.

**Comment by ganyoa** [comment]
Hi Natasha, yeah lesson07 is complete as well. I tried to initiate a pull request for 07 but see now I can't.  I'll wait for #759 to merge then request lesson07.  Thanks, Anthony

**Comment by natasha-aleksandrova** [comment]
@ganyoa if you've committed the files already then lesson 07 is part of this pull request.

we'll go ahead and review it as part of this.

File: students/ganyoa/lesson07/html_render.py [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+class Hr(SelfClosingTag):
+    tag = "hr"
+
+
+class Br(SelfClosingTag):
+    tag = "br"
+
+
+class Meta(SelfClosingTag):
+    tag = 'meta'
```
Comment (charlesrobison) [inline students/ganyoa/lesson07/html_render.py]: Looks good!

**Comment by charlesrobison** [comment]
@ganyoa - please submit your code for Lesson 7 on the course site to receive a grade

**Pull request merged**